sysboundary/sysboundary.cpp

```diff
@@ -312,91 +351,93 @@ bool SysBoundary::checkRefinement(dccrg::Dccrg<spatial_cell::SpatialCell,dccrg::
-         " at (" << mpiGrid[cellId]->parameters[CellParams::XCRD] << ", " << mpiGrid[cellId]->parameters[CellParams::YCRD] << ", " << mpiGrid[cellId]->parameters[CellParams::ZCRD] <<
-         "). Cell level = " << mpiGrid.get_refinement_level(cellId) <<
-         ", boundary level = " << innerBoundaryRefLvl << endl;
+         cout << "Failed refinement check (innerBoundary) , cellId = " << cellId << " at ("
+              << mpiGrid[cellId]->parameters[CellParams::XCRD] << ", " << mpiGrid[cellId]->parameters[CellParams::YCRD]
+              << ", " << mpiGrid[cellId]->parameters[CellParams::ZCRD]
+              << "). Cell level = " << mpiGrid.get_refinement_level(cellId)
+              << ", boundary level = " << innerBoundaryRefLvl << endl;
          return false;
       }
    }
 
    for (auto cellId : outerBoundaryCells) {
       if (cellId != INVALID_CELLID && mpiGrid.get_refinement_level(cellId) != outerBoundaryRefLvl) {
-         cout << "Failed refinement check (outerBoundary), cellId = " << cellId << 
-         " at (" << mpiGrid[cellId]->parameters[CellParams::XCRD] << ", " << mpiGrid[cellId]->parameters[CellParams::YCRD] << ", " << mpiGrid[cellId]->parameters[CellParams::ZCRD] <<
-         "). Cell level = " << mpiGrid.get_refinement_level(cellId) <<
-         ", boundary level = " << outerBoundaryRefLvl << endl;
+         cout << "Failed refinement check (outerBoundary), cellId = " << cellId << " at ("
+              << mpiGrid[cellId]->parameters[CellParams::XCRD] << ", " << mpiGrid[cellId]->parameters[CellParams::YCRD]
+              << ", " << mpiGrid[cellId]->parameters[CellParams::ZCRD]
+              << "). Cell level = " << mpiGrid.get_refinement_level(cellId)
+              << ", boundary level = " << outerBoundaryRefLvl << endl;
          return false;
       }
    }
-   
+
    return true;
 }
 
-
 bool belongsToLayer(const int layer, const int x, const int y, const int z,
-                    FsGrid< fsgrids::technical, FS_STENCIL_WIDTH> & technicalGrid) {
-   
+                    FsGrid<fsgrids::technical, FS_STENCIL_WIDTH>& technicalGrid) {
+
    bool belongs = false;
-   
+
    // loop through all neighbors (including diagonals)
    for (int ix = -1; ix <= 1; ++ix) {
       for (int iy = -1; iy <= 1; ++iy) {
          for (int iz = -1; iz <= 1; ++iz) {
-            
+
             // not strictly necessary but logically we should not consider the cell itself
             // among its neighbors.
-            if( ( ix == 0 && iy == 0 && iz == 0 ) || !technicalGrid.get(x+ix,y+iy,z+iz)) {
+            if ((ix == 0 && iy == 0 && iz == 0) || !technicalGrid.get(x + ix, y + iy, z + iz)) {
                continue;
             }
-            
-            if(layer == 1 && technicalGrid.get(x+ix,y+iy,z+iz)->sysBoundaryFlag == sysboundarytype::NOT_SYSBOUNDARY) {
+
+            if (layer == 1 &&
+                technicalGrid.get(x + ix, y + iy, z + iz)->sysBoundaryFlag == sysboundarytype::NOT_SYSBOUNDARY) {
```

Comment:
I don't see why this is split over 2 lines if line 389 above is not.

Reply:
The splitting/non-spliiting is determined by max line width, which is set to 200.